app/components/common/Header.js

```diff
@@ -1,31 +1,43 @@
 import React, { PropTypes } from 'react';
 import { Link, IndexLink } from 'react-router';
 
-const Header = () => {
+const Header = (props) => {
+  let links = [
+    { to: '/app/about', className: 'glyphicon glyphicon-exclamation-sign', label: 'About'}
+  ];
+  if(props.showAllLinks) {
```

Comment:
I guess this is because user is already authenticated, right? If that's the case what do you think or renaming this prop?

Reply:
What name do you suggest?

Reply:
What if code is changed to have something like...

```js
<Header showRestrictedOptions={this.props.isUserAuthenticated} />
```

No big deal anyways, just want to make it clear those are auth-related.

Reply:
I will take it into account, I like your suggestion.